### DISCUSSION TITLE
Can't download beatmaps with "decryption operation failed" exception

### DISCUSSION THREAD

**Discussion by Joppe27** [ppy/osu#14766] [Q&A]
When trying to download beatmaps or trying to load the beatmap listing, the operation will usually fail (but not always!) with a "decryption operation failed" error. This same error also pops up when starting lazer as an "unobserved error" but will not crash me out of the client. I've tried running lazer as administrator to make sure rights are not the issue, but that doesn't help. Logs are attached, please ask if any other network info can be of use as I'm not sure what is safe/necessary to post here.
[logs.zip](https://github.com/ppy/osu/files/7178552/logs.zip)

Note: "De opgegeven gegevens kunnen niet worden ontsleuteld" translates to "The requested data could not be decrypted".

**Comment by peppy** [comment]
This is a system level issue. You might want to google for solutions. Either something on your PC is corrupt or not running as expected.
